mappings.json

```diff
@@ -889,12 +892,18 @@
   },
   {
     "global": "Ember.String.htmlSafe",
+    "versions": {
+      "ember-source": ">= 2.18"
+    },
     "module": "@ember/template",
     "export": "htmlSafe",
     "deprecated": false
   },
   {
-    "global": "Ember.String.isHTMLSafe",
+    "global": "Ember._Template.isHTMLSafe",
```

Comment:
Since this one is for `ember-source < 2.18` I think the global should be `Ember.String.isHTMLSafe`, right?

Reply:
I closed my PR, so not sure what the final implementation will be.

Reply:
I cannot find `Ember._Template` anywhere on master. Am I missing something?